pt-BR/mail/chrome/messenger/accountCreation.dtd

```diff
@@ -57,30 +57,30 @@
      the incoming or outgoing servers, then the page that pops up will have
      this text in a button to take you back to the previous page to change
      the settings and try again. -->
-<!ENTITY changeSettings.label            "Change Settings">
+<!ENTITY changeSettings.label            "Alterar Configurações">
 <!ENTITY changeSettings.accesskey        "S">
 
-<!ENTITY pb.contactYourProvider.description "&brandShortName; can allow you to get to your mail using the provided configurations. However, you should contact your administrator or email provider regarding these improper connections.">
-
-<!ENTITY insecureServer.tooltip.title    "Warning! This is an insecure server.">
-<!ENTITY insecureServer.tooltip.details  "Click circle for more details.">
-
-<!ENTITY insecureUnencrypted.description "Your email and authentication are sent unencrypted, so your password (and your message) could easily be read by other people. &brandShortName; will let you get to your mail, but you should contact your email provider about configuring the server with a secure connection.">
-<!ENTITY insecureSelfSigned.description  "The server uses a certificate that we can't trust, so we can't be sure that someone isn't intercepting the traffic between &brandShortName; and your server. &brandShortName; will let you get to your mail, but you should really get your email provider to configure the server with a trusted certificate.">
-<!ENTITY secureServer.description        "Congratulations! This is a secure server.">
-
-<!ENTITY pb.identity.title "Identity">
-<!ENTITY pb.name.label    "Your Name">
-<!ENTITY pb.email.label   "Email Address">
-<!ENTITY pb.password.label "Password">
-<!ENTITY pb.serverinfo.title "Server Information">
-<!ENTITY pb.username.label   "User Name">
-<!ENTITY pb.incoming.label "Incoming Server">
-<!ENTITY pb.servername.label "Server Name">
-<!ENTITY pb.servertype.label "Type">
-<!ENTITY pb.serverport.label "Port">
-<!ENTITY pb.serversecurity.label "Security">
-<!ENTITY pb.outgoing.label "Outgoing Server">
+<!ENTITY pb.contactYourProvider.description "O &brandShortName; pode permitir que você obtenha seus e-mails utilizando as configurações fornecidas. Entretanto, você deve contatar seu administrador ou provedor de e-mail em relação a essas conexões inadequadas.">
+
+<!ENTITY insecureServer.tooltip.title    "Atenção! Esse servidor é inseguro.">
+<!ENTITY insecureServer.tooltip.details  "Clique para mais detalhes.">
+
+<!ENTITY insecureUnencrypted.description "O e-mail e a autenticação são enviados sem criptografia, então sua senha (e sua mensagem) poderiam ser facilmente lidas por outras pessoas. O &brandShortName; permitirá que você receba seus e-mails, mas você deveria entrar em contato com o seu provedor para configurar o servidor usando uma conexão segura.">
```

Comment:
I'll accept only the changes regarding the future tense. There's no reason to change e-mails to mensagens de correio eletrônico, and "pelo que" is not used in PTBR usually, only in PTPT.